### ISSUE TITLE
Collision between binutils and clang [Mac OS]

### ISSUE THREAD

**Issue by MarcoPolo** [tweag/rules_nixpkgs#129]
**Describe the bug**
A simple project build fails on Mac OS

**To Reproduce**
On Mac:
git clone https://github.com/MarcoPolo/bug_repro1111
bazel build //main:hello-world 

Fails.

Works on linux!

**Expected behavior**
Should build

**Environment**
 - OS name + version: MacOS Catalina
 - Version of the code: v0.7.0

**Additional context**
There should be way to set the priority of clang over binutils.

**Comment by aherrmann** [comment]
Indeed, the default Nix expression for `nixpkgs_cc_configure`
```
          with import <nixpkgs> { config = {}; overlays = []; }; buildEnv {
            name = "bazel-cc-toolchain";
            paths = [ stdenv.cc binutils ];
          }
```
causes a collision on MacOS. There are other issues with this as well. In particular nixpkgs' clang expects standard include paths in the environment, which are missing in the Bazel sandbox. You can use the setup from [`rules_haskell`](https://github.com/tweag/rules_haskell/blob/master/nixpkgs/cc-toolchain.nix) to work around this issue.

Note, https://github.com/tweag/rules_nixpkgs/pull/128 should resolve this issue as well.

**Comment by aherrmann** [comment]
I think this has been fixed by #128. Feel free to reopen if this is still an issue for you.
